graylog2-web-interface/src/views/components/widgets/EditWidgetFrame.tsx

```diff
@@ -83,6 +88,7 @@ const EditWidgetFrame = ({ children }: Props) => {
     return <Spinner text="Loading widget ..." />;
   }
 
+  const limitDuration = moment.duration(config.query_time_range_limit).asSeconds();
```

Comment:
Do we need checks here if 

1. `config` or 
2. `query_time_range_limit` are `undefined`?

The first would result in an exception, the second forces us to rely on `moment.duration` returning an empty duration if the argument is `undefined`. IMO we should make this more explicit like this:

```suggestion
  const limitDuration = config?.query_time_range_limit !== undefined ? moment.duration(config.query_time_range_limit).asSeconds() : 0;
```

Reply:
`moment.duration(undefined | null)` will return `0`. 

If `config` does not return Search Config, then we are already having issues everywhere else with searching because we will not have the necessary data.

If `query_time_range_limit` also does not return something, then we should define this as a default in the SearchConfigStore to always return `0` if it does not exist.

I do agree that there should be some checking though so added the optional chaining to the object

`const limitDuration = moment.duration(config?.query_time_range_limit).asSeconds();`

Maybe something like `const limitDuration = moment.duration(config?.query_time_range_limit).asSeconds() ?? 0;` would be good?

Reply:
The last option would be okay for me too.